RevenueCatUI/CustomerCenter/Views/ManageSubscriptionsView.swift

```diff
@@ -50,26 +69,32 @@ struct ManageSubscriptionsView: View {
     }
 
     var body: some View {
-        if #available(iOS 16.0, *) {
-            content
-                .navigationDestination(isPresented: .isNotNil(self.$viewModel.feedbackSurveyData)) {
-                    if let feedbackSurveyData = self.viewModel.feedbackSurveyData {
-                        FeedbackSurveyView(feedbackSurveyData: feedbackSurveyData,
-                                           customerCenterActionHandler: self.customerCenterActionHandler)
+        content
+            .background(
+                Group {
+                    if #available(iOS 16.0, *), let navigationPath = navigationPathBinding {
+                        content
```

Comment:
Is it necessary to re-render `content` in the background on `content`?

It might be more efficient to place these modifiers on something that we know won't have a lot of rendering complexity. I'll often use something like `Color.clear` for things like this, and set the opacity to 0 to be safe